libs/redis/langchain_redis/chat_message_history.py

```diff
@@ -107,7 +107,8 @@ def _ensure_index(self) -> None:
         try:
             self.redis_client.ft(self.index_name).info()
         except ResponseError as e:
-            if str(e).lower() == "unknown index name":
+            msg = str(e).lower()
```

Comment:
@bsbodden eventually we should wrap this kind of stuff in RedisVL -- also fyi `index.info()` from rvl wraps this piece for you. We also have `index.exists()` :)